src/layout/rect/iter.rs

```diff
@@ -68,6 +73,26 @@ impl Iterator for Columns {
     }
 }
 
+impl DoubleEndedIterator for Columns {
+    /// Retrieves the previous column within the `Rect`.
+    ///
+    /// Returns `None` when there are no more columns to iterate through.
+    fn next_back(&mut self) -> Option<Self::Item> {
+        if self.current_column_back <= self.rect.x
+            || self.current_column_back <= self.current_column
```

Comment:
```suggestion
        if self.current_column_back <= self.current_column
```

(and change to `<` if needed)

Reply:
Likewise.